app/models/breakeven_table_question.rb

```diff
@@ -85,6 +85,15 @@ def initialize(breakeven)
   #   periods: 4,
   #   units: 'Months'
   # }
+
+  PERIOD_TYPES = %i(days weeks months quarters years).freeze
```

Comment:
Basically, it's to protect from constants getting mutated accidentally. It seems that rubocop also thinks it's a good idea. http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant